src/ui/src/features/traces/traces.css

```
@@ -5,6 +5,144 @@
+}
+
+.trace-heatmap-ylabel {
+  text-anchor: end;
+}
+
+.trace-heatmap-summary {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)